packages/typescript-checker/src/grouping/node.ts

```diff
@@ -0,0 +1,38 @@
+import { Mutant } from '@stryker-mutator/api/src/core';
+
+export class Node {
```

Comment:
Is `Node` a clear enough name? What about `TSFileNode` or maybe `TSFileGraph`? 

Can we add a description above it to explain why this class exists? I assume it is because TS only provides a `getAllDependencies(file)` API, and we need a 2-way dependency graph.